src/server/string_family.cc

```diff
@@ -1512,6 +1512,67 @@ void StringFamily::ClThrottle(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+struct StringValue {
```

Comment:
is it to save the allocation for future?

Reply:
yes, future<T> has some overhead by default